pkg/lang/frontend/starlark/install/install.go

```diff
@@ -187,9 +187,10 @@ func ruleFuncVSCode(thread *starlark.Thread, _ *starlark.Builtin,
 func ruleFuncConda(thread *starlark.Thread, _ *starlark.Builtin,
 	args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
 	var name, channel *starlark.List
+	var envFile starlark.String
 
 	if err := starlark.UnpackArgs(ruleConda,
-		args, kwargs, "name", &name, "channel?", &channel); err != nil {
+		args, kwargs, "name", &name, "channel?", &channel, "envFile?", &envFile); err != nil {
```

Comment:
How about `env_file` here? We always use lower case in starlark part

Reply:
def build():
    install.conda_packages(name = [
        "pytorch",
        "cudatoolkit=11.3",
    ], channel= ["pytorch"])
    base(os="ubuntu20.04", language="python3.8",envFile="env.yaml")
    install.python_packages(name = [
        "flask"
    ])

Reply:
ok ，I try to lowercase

Reply:
envFile="env.yaml" should be in `install.conda_packages` instead of `base`

Reply:
def build():
    install.conda_packages(name = [
        "pytorch",
        "cudatoolkit=11.3",
    ], channel= ["pytorch"],env_file="env.yaml")
    base(os="ubuntu20.04", language="python3.8")
    install.python_packages(name = [
        "flask"
    ])
The above configuration file is wrong

Reply:
Lowercase parameter names also report an error
![image](https://user-images.githubusercontent.com/22475606/181416452-55d7b29f-04d6-49e2-a99b-7525ca2df529.png)

Reply:
Please use ./bin/envd instead of the envd you installed to run.

Reply:
ok

Reply:
#0 0.186 mkdir: cannot create directory '/opt/conda/pkgs': File exists                                                                                          
------
ERRO[2022-07-28T04:33:27Z] failed to load docker image: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.41/images/load?quiet=1": context canceled  tag="conda:dev"
error: process "mkdir /opt/conda/pkgs" did not complete successfully: exit code: 1

Reply:
Please clean the cache and give another try:

```
./bin/envd prune --all
```

Reply:
still the same error

Reply:
Could you show the full debug log here?

Reply:
I added the -p parameter to the mkdir command

Reply:
I did not find where you add it. Could you push the code?

Reply:
At present, the program can run normally, but the dependencies in the yaml file are not installed

Reply:
> I did not find where you add it. Could you push the code?

The code has been pushed

Reply:
yaml file can be downloaded normally